core/src/components/datetime/test/presentation/e2e.ts

```diff
@@ -13,3 +13,37 @@ test('presentation', async () => {
     expect(screenshotCompare).toMatchScreenshot();
   }
 });
+
+describe('presentation: time', () => {
+
+  let page: E2EPage;
+
+  beforeEach(async () => {
```

Comment:
Can we also add tests to the picker column? Some ideas:

1. Check that scrolling the picker emits ionChange.
2. Check that setting `value` prop does not emit ionChange.